client/adventureMap/AdventureMapInterface.cpp

```diff
@@ -172,6 +173,8 @@ void AdventureMapInterface::dim(Canvas & to)
 		if (!casted && !window->isPopupWindow())
 		{
 			int backgroundDimLevel = settings["adventure"]["backgroundDimLevel"].Integer();
+			if(isHotseatMessage)
+				backgroundDimLevel = 255;

```

Comment:
why not moving backgroundDimLevel to class fields and set it to 255/settings from onHotseatWaitStarted/onPlayerTurnStarted methods?

Reply:
Would also be possible. But I think it's more understandable like this? Or not?

Reply:
surely it's personal preference. Extra variable which is used only to control another variable from another method a bit confusing. Controlling dimming directly in onHotseatWaitStarted states clearly, that you want to dim screen on hotseat message

Reply:
Okay, convinced. Are fewer lines. :)